rcnn/common.hpp

```diff
@@ -77,23 +77,32 @@ static inline int read_files_in_dir(const char *p_dir_name, std::vector<std::str
 }
 
 static inline cv::Mat preprocessImg(cv::Mat& img, int input_w, int input_h) {
-    int w, h, x, y;
+    int w, h;
+    float x, y;
     float r_w = input_w / (img.cols*1.0);
     float r_h = input_h / (img.rows*1.0);
     if (r_h > r_w) {
         w = input_w;
         h = r_w * img.rows;
-        x = 0;
+        x = 0.0;
         y = (input_h - h) / 2;
```

Comment:
the problem still exists though y is float, because input_h, h are all int.
one way is y = (input_h - h) / 2.f;

Reply:
looks like this function haven't be called

Reply:
No, this function will be called by {bool Int8EntropyCalibrator2::getBatch(void* bindings[], const char* names[], int nbBindings)} as mentioned by you. You can check it.